packages/chains/src/chainId.ts

```diff
@@ -11,6 +11,7 @@ export enum ChainId {
   POLYGON_ZKEVM_TESTNET = 1442,
   ARBITRUM_ONE = 42161,
   ARBITRUM_GOERLI = 421613,
+  SCROLL = 534352,
```

Comment:
let's do not add scroll for now. We can sync that from develop after we merge the scroll branch